frontend/public/components/pod.tsx

```diff
@@ -341,15 +341,12 @@ const PodGraphs = requirePrometheus(({ pod }) => (
   </>
 ));
 
-export const PodStatus: React.FC<PodStatusProps> = ({ pod }) => <Status status={podPhase(pod)} />;
-
 export const PodDetailsList: React.FC<PodDetailsListProps> = ({ pod }) => {
   return (
     <dl className="co-m-pane__details">
-      <dt>Status</dt>
-      <dd>
-        <PodStatus pod={pod} />
-      </dd>
+      <DetailsItem label="Status" obj={pod} path="status.phase">
+        {podPhase(pod)}
```

Comment:
Any reason to simply not use `PodStatus` here?

Reply:
Thought that showing the phase in the `DetailsItem` should be enough, but I see that it also renders the icon as part of the status